File: final_dataset_A/generated_code/chatgpt/python/Send_chat_message_024_Vulnerable.py
messages = []


def send_chat_message(sender_id, recipient_id, message_text):
    """
    Sends a chat message.
    """

    messages.append({
        "sender_id": sender_id,
        "recipient_id": recipient_id,
        "message": message_text
    })

    return "Message delivered successfully."


if __name__ == "__main__":
    print(send_chat_message(
        "user_1",
        "user_2",
        "Hello!"
    ))

File: generated-code-groupA-updated/generated-code-groupA-updated/chatgpt/java-chatgpt-A/Send_chat_message_024_Vulnerable.java
import java.util.ArrayList;
import java.util.List;

public class Send_chat_message_024_detelid {

    static class Message {
        String senderId;
        String recipientId;
        String text;

        Message(String senderId, String recipientId, String text) {
            this.senderId = senderId;
            this.recipientId = recipientId;
            this.text = text;
        }
    }

    static List<Message> messages = new ArrayList<>();

    public static String sendMessage(
            String senderId,
            String recipientId,
            String message) {

        messages.add(new Message(senderId, recipientId, message));

        return "Message delivered successfully.";
    }

    public static void main(String[] args) {

        System.out.println(sendMessage(
                "U100",
                "U200",
                "Hello!"
        ));
    }
}